src/r/utils/setStyle.js

```diff
@@ -12,7 +12,7 @@ import parseValue from './parseValue';
  * @returns {*}
  */
 const setStyle = (element, property, value) => {
-  if (typeof value === 'undefined') return false;
+  if (typeof value === 'undefined') return undefined;
```

Comment:
Can't you just do a blank `return`?

Reply:
not in accordance with consistent-return lint rule :)

Reply:
Hmm okay.

Reply:
Shouldn't we come up with something for these cases? Like always returning `null` instead of `undefined` or something like that.

Reply:
yeah good idea. I changed it to undefined here, because that's what the test expected